src/pages/sign-up/SignUp.tsx

```diff
@@ -28,6 +28,7 @@ const schema = Yup.object().shape({
     .matches(/^[aA-zZ\s]+$/, 'Please enter valid name')
     .required('*Lastname is required'),
   password: Yup.string().required('*Password is required'),
+  phoneNumber: Yup.number().min(100000000000, 'Phone number must be 12 digits'),
```

Comment:
1. Do we need to make phone number mandatory . Could you pls confirm with Simy and do change here if yes
2. This validation won't work in country code right? Don't we need a validation for country code and number of   digits? (You can also confirm that with Simy)
3. In TECH-673 , there is one more change like  add a space between text 'PhoneNumber' -> Phone Number. Can you pls do that change too?

Reply:
Hello yes I did the changes it's not reflecting due to some issues in my git so i'll amend and do these changes again
For the country code in the I think they have planned to do it later as the excel shared by QA there it mentioned. I'll check with Simi again. And for this using Yup it is making mandatory without using .required also.